data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,513 @@
+				not "Hai Legacy 1: deferred"
+			`Do you want to head to the coordinates as the hooded figure suggested?`
+			choice
+				`	(Sure.)`
+					goto go
+				`	(Maybe another time.)`
+					defer
+				`	(Never.)`
+					decline
+			label first
+			`As you enter the shipyard you quickly get approached by a hooded figure. They come to address you discreetly, and, seeing you stop to listen, they take your arm and make you follow them. "Keep walking," they say, taking a look around the way someone looking for a new ship would, pointing at one with little conviction before addressing you again. "My associates and I have a lucrative business proposition for you. If you are interested go to these coordinates." They give you a piece of paper with the location, a beach on the other side of the planet, before quickly fading into the crowd. Will you go there?`
+			choice
+				`	(Sure.)`
+				`	(Maybe another time.)`
+					defer
+			label go
+			`	As you ponder how to make your way to the rendezvous point, you notice a convenient train ticket in your pocket, with instructions on how to find your way to the destination. You proceed to take the train, and then have to walk for hours to get the rest of the way there.`
+			`	You look around you, double checking you are indeed at the right coordinates, for except a cave, vegetation, and, of course, the beach, you cannot see much of interest in the vicinity. You can't help but keep your hand close to your gun, feeling safer with its presence.`
+			`	After waiting for what seems like an eternity you spot two hooded figures approaching you, whispering amongst themselves. You start to doubt whether or not this was a good idea, but it is probably too late to back down now. They come to address you, their face somehow impossible to glimpse at, even below the hoodie. "Hello, <last>," they say at the same time, almost in a comical way, removing some of the tension you felt until now. Then they look at each other, nod, and one of them addresses you in a more serious tone. "We are sorry for all of these complications, but the truth is, we cannot afford to take risks. You see, we simply do not know how you might react. Furthermore, it is best for all if you do not come to know us."`
+			choice
+				`	"Tell me more."`
+					goto tellme
+				`	"How do you know me?"`
+					goto know
+				`	(Run away).`
+			`	They seem surprised at your sudden change of mind, but make no effort to stop you.`
+			label trainback
+			`	You proceed to head to the nearest spaceport by getting yourself another train ticket, without any incidents.`
+				decline
+			label know
+			`	They grin, "You need to become known by the Hai in order to land on our homeworld."`
+			`	"We know you have provided help, by doing various jobs."`
+				to display
+					"reputation: Hai (Unfettered)" >= -1250
+			`	"Do not think we are unaware of your skirmishes with the Unfettered," they add with a serious tone.`
+				to display
+					"reputation: Hai (Unfettered)" < -1250
```

Comment:
Is this supposed to be listed twice in this section? It seems appropriate only for this last answer.

Reply:
Ah! I just missed the different signs, was focused on the matching values... but that makes sense as a switchover point :)